POUF1.md

```diff
@@ -0,0 +1,442 @@
+
+The image repository hosts the RootMetadata, TargetsMetadata, SnapshotMetadata, TimestampMetadata, and images. The filenames for the metadata files are <role>.der where role is the associated role (root, targets, snapshot, or timestamp). The filenames for the images are specified in the custom field of targets metadata on the Image repository as described in the [Uptane Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#targets_images_meta). The url of the hosted directory will be loaded onto primary ECUs and full verification secondary ECUs at the time of manufacture.
+
+# Operations    
+Of course, as the system implements Uptane, the reference implementation follows the requirements of the Uptane Standard. In addition to the required features, the reference implementation follows most of the SHOULDs of the standard in order to demonstrate how Uptane is expected to operate. The implementation details and rationale behind any SHOULDs and MAYs that are implemented are described in this section.
+
+The Primary ECU in this POUF is also the ECU that communicates with the director repository, image repository, and time server. This simplifies the reference implementation by condensing all the requirements for external communication and initial validation into a single ECU.
+
+To access a reliable current time as needed for the [In Vehicle Implementation Requirements](https://uptane.github.io/uptane-standard/uptane-standard.html#in-vehicle-implementation-requirements), ECUs in this POUF use a time server that produces a time attestation for each update cycle. The Primary ECU sends the time server nonce tokens from each ECU, then the time server returns a time attestation that contains a signed the current time with all the tokens, as described in the [Deployment Considerations](https://uptane.github.io/deployment/introduction.html). The ASN.1 details of the time server are listed in {Formats}. The time server is used by the reference implementation as a self contained way to ensure secure access to the current time.
+
+Additionally, this POUF supports custom delegated targets roles and terminating delegations, features described as optional in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#targets_role_delegations). As explained in the Standard, terminating delegations are not used by default. To use custom roles, the Targets role on the Image repository sets up a delegation. Delegations used in this POUF must support multi-role delegations, as explained in [TAP 3](https://github.com/theupdateframework/taps/blob/d0818e580c322815a473520f2e8cc5f5eb8df499/tap3.md). In addition, delegation filenames are listed using an enumerated list that supports [wildcards](https://uptane.github.io/uptane-standard/uptane-standard.html#delegations_meta). Delegations are supported in the reference implementation to demonstrate how they can be used to establish a chain of trust for specific images.
```

Comment:
Can wildcards be everywhere?  What wildcard characters are supported?  Can you use them in directories?  How do they match directory paths (e.g., does "a*.txt" match "apple/file.txt")?

Reply:
I clarified that these are shell-style wildcards. I can add more description if needed.